src/service/operation-id-service.js

```diff
@@ -102,24 +102,31 @@ class OperationIdService {
     }
 
     async cacheOperationIdData(operationId, data) {
-        this.logger.debug(`Caching data for operation id: ${operationId} in file`);
-        const operationIdCachePath = this.fileService.getOperationIdCachePath();
-
-        await this.fileService.writeContentsToFile(
-            operationIdCachePath,
-            operationId,
-            JSON.stringify(data),
-        );
-
-        this.memoryCachedHandlersData[operationId] = { data, timestamp: Date.now() };
+        try {
```

Comment:
Did you check if we are handling errors when this method is called in other places. Maybe we expect that error is thrown and now that you are only logging the whole flow could be wrong. Be careful when you are handling the error and just logging in catch block. Also add more context where error happened don't only log error.message

Reply:
you were right, i changed it, thanks